Week3/week3HomeWork/Exercise3/fortuneTeller.js

```diff
@@ -0,0 +1,39 @@
+const numChildren = [1, 2, 3, 4, 5];
+const partnerNames = ["Jessica", "Wilma", "Nadia", "Susan", "Angelina"];
+const locations = ["Syria", "London", "Turkey", "Netherland", "Franca"];
+const jobs = ["accountant", "brogrammer", "salesman", "businessman", "manager"];
+
+function tellFortune(child, partener, location, job) {
+    const numChildren = [1, 2, 3, 4, 5];
+    const partnerNames = ["Jessica", "Wilma", "Nadia", "Susan", "Angelina"];
+    const locations = ["Syria", "London", "Turkey", "Netherland", "Franca"];
+    const jobs = [
```

Comment:
Why have you also added these arrays inside the function? (and outside of it)

Reply:
that was fault from me ....